library/src/main/java/com/alamkanak/weekview/WeekView.java

```diff
@@ -1085,6 +1096,9 @@ private void drawEventTitle(WeekViewEvent event, RectF rect, Canvas canvas, floa
         int availableWidth = (int) (rect.right - originalLeft - mEventPadding * 2);
 
         // Get text dimensions.
+        if(textColorPicker != null) {
```

Comment:
This is the desired behavior I think, as each event can require a different textColor (e.g. events with a dark background gets white text and events with a white background get black text)

If it is set in the init() function, all events will have the same text color.